drivers/wifi/esp_at/esp.c

```diff
@@ -726,9 +742,20 @@ static void esp_mgmt_scan_work(struct k_work *work)
 	if (ret < 0) {
 		goto out;
 	}
-	ret = esp_cmd_send(dev, cmds, ARRAY_SIZE(cmds), "AT+CWLAP",
+	ret = esp_cmd_send(dev,
+#if defined(CONFIG_WIFI_ESP_AT_SCAN_PASSIVE)
+			   cmds, ARRAY_SIZE(cmds), "AT+CWLAP=,,,1,,",
+#else
+			   cmds, ARRAY_SIZE(cmds), "AT+CWLAP",
+#endif
```

Comment:
I'd rather have a macro like:
```
#if defined(CONFIG_WIFI_ESP_AT_SCAN_PASSIVE)
#define ESP_CMD_CWLAP "AT+CWLAP=,,,1,,"
#else
#define ESP_CMD_CWLAP "AT+CWLAP"
#endif
```
so that the function is a bit less noisy.

Reply:
Done